### ISSUE TITLE
Working on my mame folders.. REALLY laggy moving folders and files.

### ISSUE THREAD

**Issue by tacogerbil** [files-community/Files#13389]
### Description

Just moving files and folders around.  Have 2 windows open, just transferring files and folders within the same folder..
(I do have some tabs open to other directories/drives.. some on my NAS but not actively using those).
While moving a folder (on a local drive) that had one file inside of it.  The directory it was in has about 1200 files/folders it.  It started to lag and just crashed and re-opened and prompted me to report the issue.
Note:  Within the folder that has 1238 files I am going INTO specific folders, placing a file INTO IT and then backing out to the root folder that has 1238 files/folder repeatedly.

### Steps To Reproduce

Moving files and folders within windows on local SATA HDD drives

### Requirements

- e.g. Fix crash that occurs when...
Don't lag out and crash when moving files

### Files Version

2.5.29.0

### Windows Version

10.0.22621.2283

### Log File

[debug.log](https://github.com/files-community/Files/files/12643478/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, what layout are you using?

**Comment by tacogerbil** [comment]
I use these 2 windows.  
![image](https://github.com/files-community/Files/assets/12961907/c357bfec-d410-4eea-9b5c-587644a893f2)
I move a folder from the right side into a folder on the left.  I first open the folder on the left before moving the file/folder into it.  Once the file/folder is moved I back out again.  Rinse and repeat :)
They are all ZIP and CHD files.

**Comment by tacogerbil** [comment]
I started to work on a large amount of files (mostly png's, jpg's and some zips.. over 1 tb on LOCAL drives) and it's crashed a few times the same way.  It gets REALLY laggy when dealing with large amounts of files.  
I'm realizing that when I go to drag a file/folder from one directory to another (using two separate Files windows as in the image above) and I cross the file/folder OVER the icon of a "favorite" in the Favorites/Drives column... Files very quickly assumes I'm trying to drag the file into that folder.  I'm not pausing over the folder shortcut, rather dragging across it in order to get into the folder I want that is on the separate window.  This action causes the directory that I was originally dragging from to change to the directory that was "sensed" I was dragging over, which in turn causes me to have to back out of the new directory and back into the previous one.  This is where I started to notice that it reloads that whole folder again and it's slow... it starts to lag.  If I do anything else within these seconds of waiting (even on a different Files window) it pauses intermittently and will sometimes crash and re-open.  
I don't have any extra setting on it like displaying file sizes or anything like that.  Pretty much left at default.
I don't see this in regular explorer.exe (at least to this extent) with large amounts of files.
